components/scream/src/physics/shoc/tests/shoc_pblintd_check_pblh_tests.cpp

```diff
@@ -80,21 +80,27 @@ struct UnitWrap::UnitTest<D>::TestPblintdCheckPblh {
   static void run_bfb()
   {
     PblintdCheckPblhData f90_data[] = {
-      // TODO
+      PblintdCheckPblhData(36,  72, 73),
+      PblintdCheckPblhData(72,  72, 73),
+      PblintdCheckPblhData(128, 72, 73),
+      PblintdCheckPblhData(256, 72, 73),     
     };
 
     static constexpr Int num_runs = sizeof(f90_data) / sizeof(PblintdCheckPblhData);
 
     // Generate random input data
     // Alternatively, you can use the f90_data construtors/initializer lists to hardcode data
     for (auto& d : f90_data) {
-      d.randomize();
+      d.randomize({ {d.check, {1, 1}} });
```

Comment:
I think `val != 0 -> true`. So this most likely results in all values true. Maybe hardcode true/false in a loop over shcol? Not sure what is best..

Reply:
Never mind, I guess this is how PhysicsData treats randomizing bools. Did not realize that.